### DISCUSSION TITLE
Deploying Nuxt3 to Firebase - NITRO_PRESET is not recognized

### DISCUSSION THREAD

**Discussion by AloisSeckar** [nuxt/nuxt#18195] [Questions] [answered]
I am trying to follow this tutorial https://v3.nuxtjs.org/docs/deployment/firebase , working on Windows 10 Home, using VS code.

0a) Set up brand new project following instructions in https://v3.nuxtjs.org/getting-started/installation => got working localhost:3000 site

0b) Created _&lt;my-firebase&gt;_ project on Firebase and upgraded it to _Blaze_ plan right away.

1) Created `.firebaserc` file
```
{
  "projects": {
    "default": "<my-firebase>"
  }
}
```

2) Ran
```
npm install -D firebase-admin firebase-functions firebase-functions-test
```

3) Created `firebase.json` file
```
{
    "functions": { "source": ".output/server" },
    "hosting": [
      {
        "site": "<my-firebase>",
        "public": ".output/public",
        "cleanUrls": true,
        "rewrites": [{ "source": "**", "function": "server" }]
      }
    ]
}
```

4) Tried to run `NITRO_PRESET=firebase npm run build` in VS terminal, but ended up with following error message:

```
NITRO_PRESET=firebase : The term 'NITRO_PRESET=firebase' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling o
f the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ NITRO_PRESET=firebase npm run build
+ ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (NITRO_PRESET=firebase:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Certainly I am missing something in my setup, but can someone help me figuring what it is? Do I have to install Nitro somehow on my machine or add something to PATH to be able to use it? Was unable to google anything useful so far. Thanks for any help and sorry if I am missing some info required to help me in my post.

**Comment by danielroe** [comment, marked as answer]
Looks like you are running that on Windows. You'll likely need to run, for example:
```
npx cross-env NITRO_PRESET=firebase npm run build
```

(Or you can use a bash-compatible shell.)

**Reply by AloisSeckar** [reply]
Yay, that was it. `npx cross-env` was the missing part.  Thanks a lot.
